### PR TITLE
First Draft of Updates to Element Voiding CTs

### LINKED ISSUE #214: Earthworks Cut Template & Adjustments to Element Opening Templates
## Description & Tasks
Addition of Earthworks cut template by the conversion of Element Voiding into a Group of Earthwork Cuttings and Element Openings.

- [x] New Earthworks Cuttings Template
- [x] New Element Openings Template (derived from original Element Voiding Template)
- [x] Coversion of Element Voiding to a Template Grouping

### Discussion Points

1. Structure of new Templates - Agree/Disagree?
2. Other Issues?

### Links
[Miro Element](https://miro.com/app/board/o9J_l5rJCLk=/?moveToWidget=3074457362318627197&cot=10)
[Miro Task Card](https://miro.com/app/board/o9J_l5rJCLk=/?moveToWidget=3074457362318412242&cot=14)

### Decisions 
TBC

### PR DESCRIPTION
**Pull request by AlexBrad1eyCT**
Fixes #214 

voiding converted to group and CTs created for earthworks cut, openings and voiding features

### CODE REVIEW

File: IFC4x3/Templates/Object Composition/Element Voiding/Earthworks Cuttings/Documentation.md [hunk 1]
```diff
@@ -0,0 +1,3 @@
+Earthworks Cuttings represent a voiding of an existing or constructed Earthen element (referred to as the voided element)  usually represented by a subtype of _IfcGeotechnicalElement_ or an _IfcEarthworksFill_, t. the _IfcEarthworksCut_ semantically represents the work of removing/excavating material from the voided element, and the associated geometrical representation is the resulting void created. A_IfcEarthworksCut_ May optionally be filled (fully or partially) with another element such as a foundation, earthen fill or foundation utilising the _IfcRelFillsElement_ relationship.
```
Comment (pjanck) [inline IFC4x3/Templates/Object Composition/Element Voiding/Earthworks Cuttings/Documentation.md:1]: What's a `t.`?

Comment (pjanck) [inline IFC4x3/Templates/Object Composition/Element Voiding/Earthworks Cuttings/Documentation.md:1]: You may want to revise `A_IfcEarthworksCut_ May`

File: IFC4x3/Templates/Object Composition/Element Voiding/Earthworks Cuttings/Documentation.md [hunk 2]
```diff
@@ -0,0 +1,3 @@
...
+
+The resulting geometry of the relationship when applied is the result of CSG operations where the IfcEarthworksCut representation is subtracted from the voided element representation.
```
Comment (pjanck) [inline IFC4x3/Templates/Object Composition/Element Voiding/Earthworks Cuttings/Documentation.md:3]:
```suggestion
The resulting geometry of the relationship, when applied, is the result of CSG operations where the _IfcEarthworksCut_ representation is subtracted from the voided element representation.
```

**Review by pjanck — CHANGES_REQUESTED** [review]
Missing on all entries:  `Identification`.

File: IFC4x3/Templates/Object Composition/Element Voiding/Earthworks Cuttings/Documentation.md (hunk 1 above)
Comment (AlexBrad1eyCT) [reply IFC4x3/Templates/Object Composition/Element Voiding/Earthworks Cuttings/Documentation.md]: oo ehh random sorted.

Comment (AlexBrad1eyCT) [reply IFC4x3/Templates/Object Composition/Element Voiding/Earthworks Cuttings/Documentation.md]: sorted

File: IFC4x3/Templates/Object Composition/Element Voiding/Earthworks Cuttings/Documentation.md (hunk 2 above)
Comment (AlexBrad1eyCT) [reply IFC4x3/Templates/Object Composition/Element Voiding/Earthworks Cuttings/Documentation.md:3]: thanks for catching that one!

**Review by pjanck — APPROVED** [review]
lftm

**Comment by AlexBrad1eyCT** [comment]
@LeeGregory12d, @jmirtsch as providers of Earthworks unit tests your review would be appreciated!

**Pull request merged**